pkg/keys/ed25519.go

```diff
@@ -83,12 +83,21 @@ func GenerateEd25519Key() (*ed25519Signer, error) {
 	}, nil
 }
 
+func Ed25519Key(keyValue Ed25519PrivateKeyValue) *ed25519Signer {
```

Comment:
nit: constructor convention
```suggestion
func NewEd25519Signer(keyValue Ed25519PrivateKeyValue) *ed25519Signer {
```

Reply:
+1, potentially also `Import`?

Reply:
You mean `Import` instead of `New` @asraa ? Making sure I understand

Reply:
Yeah, but I'm not picky with wording :)